src/nvim/normal.c

```diff
@@ -3586,13 +3586,16 @@ void scroll_redraw(int up, long count)
  */
 static void nv_zet(cmdarg_T *cap)
 {
-  long n;
+  int n;
   colnr_T col;
   int nchar = cap->nchar;
   long old_fdl = curwin->w_p_fdl;
   int old_fen = curwin->w_p_fen;
   bool undo = false;
 
+  assert(p_siso <= INT_MAX);
+  int p_siso_l = (int)p_siso;
```

Comment:
:+1: I just got it wrong.
This was a little different than usual. Intention was not to make a local copy to avoid analyzer thinking global variable can change value, but to make a local copy with a different type (long --> int) to avoid casting/checking in several places. I was going to call it `p_siso_int`, or `p_siso_i` to highlight that intention. Then, I thought using normal convention (prefix "l_") would be enough, and it wasn't worth introducing a new convention for type-changed vars. But I used the wrong form.
So, I can use the "l_" prefix, or I can use a prefix "int_" or "i_", or maybe "l_i_". What do you think?